services/github-backup/default.nix

```diff
@@ -0,0 +1,59 @@
+{ config, pkgs, ... }:
+{
+  sops.secrets.github-backup-token = { }; #TODO: add secret to sops
+
+  systemd.services.github-backup = {
+    script = ''
+      GHORG_GITHUB_TOKEN="$(<$CREDENTIALS_DIRECTORY/github-backup-token)" \
+        ${pkgs.lib.getExe pkgs.ghorg} clone nix-community \
+        --backup \
+        --clone-wiki \
+        --concurrency 2 \
+        --exclude-match-regex '^nixpkgs$' \
+        --exclude-match-regex '^nix$' \
+        --path /var/lib/github-backup \
```

Comment:
Since this is public information, is there a way to make this backup public accessible?

Reply:
Do you mean serving this folder with nginx or something else?

Reply:
Something like that. I just don't know how easy it is to download the backup over nginx.

Reply:
Can you explain why you want the backups to be publicly accessible? My understanding was running a mirror wasn't the objective, it was just a way of being able to have backups?

Reply:
We could expose the storage box from `web01` if we want it publicly accessible.

Reply:
with its own subdomain.

Reply:
> Can you explain why you want the backups to be publicly accessible? My understanding was running a mirror wasn't the objective, it was just a way of being able to have backups?

I'd still like an answer to this?